source/documentation/monitoring-alerts/create-edit-alerts-using-prometheus.md

```diff
@@ -84,6 +84,45 @@ If you have not yet set up a receiver or would like to set up additional receive
 
 If you need additional PagerDuty licences contact the Reliability Engineering Autom8 team using the [#re-autom8 Slack channel][14].
 
+### Testing your alerts
+
+Once your alerts appear in [Prometheus][1] and you have configured PagerDuty
+to use Prometheus events to trigger incidents or suppress alerts on a service,
+you can test your alert out using [`amtool`][15].
+
+You can use `amtool` to create an alert in AlertManager. This will send a FIRING
+event to PagerDuty, which (if all is configured correctly) should trigger an
+incident in PagerDuty.
+
+Connect to the VPN, then run the following command to create an alert:
```

Comment:
Not blocking:

This line should probably explain why, but it's not that important, ie

"alertmanager is only accessible from the GDS office IP range, so you may need to use the VPN"

Reply:
Thanks, I'd forgotten about the office, I've added a note.